system-tests/lib/cre/don/jobs/jobs.go

```diff
@@ -31,6 +32,10 @@ func Create(offChainClient deployment.OffchainClient, don *devenv.DON, flags []s
 				defer cancel()
 				_, err := offChainClient.ProposeJob(ctx, jobReq)
 				if err != nil {
+					// Workflow specs get auto approved
+					if strings.Contains(err.Error(), "cannot approve an approved spec") {
```

Comment:
can we scope it more narrowly? e.g. only specs of workflow type?

Reply:
Added a TODO :)